cluster/test-e2e.sh

```diff
@@ -14,20 +14,156 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Tests a running Kubernetes cluster.
-# TODO: move code from hack/ginkgo-e2e.sh to here
```

Comment:
The TODO is to move the code, but I only see additions (and not deletions from the other file). Is there a reason we can't delete the corresponding code in ginkgo-e2e.sh? If not, we at least need to add a TODO there around the corresponding block to clean it up (ideally with information about what is preventing us from doing it now).

Reply:
great,  will try